Adyen/UI/Form/Items/Text/FormTextItem.swift

```diff
@@ -46,6 +52,21 @@ open class FormTextItem: FormValueItem<String, FormTextItemStyle>, ValidatableFo
     public func isValid() -> Bool {
         validator?.isValid(value) ?? true
     }
+    
+    /// The formatted text value.
+    @Observable("") internal var formattedValue: String
+
+    // MARK: - Private
+
+    private func publish(newValue: String) {
```

Comment:
Shall we rename this to `sanitizeAndPublish` or something as we are altering the `value` in its own `set` ?

Reply:
What do you think about `private func publishSanitized(value: String)`?

Or maybe `private func publishTransformed(value: String)` since it's sanitizing and formatting?